sapi/fpm/php-fpm.conf.in

```diff
@@ -125,6 +125,18 @@
 ; Default value: 10
 ;systemd_interval = 10
 
+; Bind the master process to a cpu set.
+; The value can be one cpu id, a range or a list thereof.
+;
+; Default Value: not set
+; Valid syntaxes are:
+;       process.cpu_list = "cpu id"                 - to bind to a single core
+;       process.cpu_list = "min cpu id-max cpu id"  - to bind to a core range
+                                                      from minimum cpu id to max
+;       process.cpu_list = "[min cpu id-max cpu id],[min cpu id-max cpu id],..."
+                                                    - to bind to multiple ranges
+                                                      separated by a comma
```

Comment:
This is actually very confusing description and a bit incorrect. Could you maybe use BNF to describe and keep some examples. I didn't mean to replace the examples completely but to add some generic description.